tools/update-schema

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+
+from __future__ import annotations
+
+import argparse
+import difflib
+import json
+import subprocess
+import sys
+from pathlib import Path
+from typing import Any
+
+from fmu.dataio._definitions import SchemaBase
+from fmu.dataio._model import FmuResultsSchema
+from fmu.dataio._products import InplaceVolumesSchema
+
+GREEN = "\033[32m"
+RED = "\033[31m"
+YELLOW = "\033[93m"
+NC = "\033[0m"
+BOLD = "\033[1m"
+PASS = f"[{BOLD}{GREEN}✔{NC}]"
+FAIL = f"[{BOLD}{RED}✖{NC}]"
+INFO = f"[{BOLD}{YELLOW}+{NC}]"
+
+SCHEMAS = [
+    FmuResultsSchema,
+    InplaceVolumesSchema,
+]
```

Comment:
Here and in the tests we are collecting a list of all (two) the schemas to iterate over them. There should probably be a place in the code to collect them, but I think waiting for the next one to see where that should be might land it in a better spot (maybe?)